translation/source/contact.xml

```diff
@@ -51,7 +51,8 @@
   <string name="fideMate">According to the FIDE Laws of Chess §6.9, if a checkmate is possible with any legal sequence of moves, then the game is not a draw</string>
   <string name="knightMate">It is possible to checkmate with only a knight or a bishop, if the opponent has more than a king on the board.</string>
   <string name="noRatingPoints">No rating points were awarded</string>
-  <string name="ratedGame">Make sure you played a rated game. Casual games do not affect the players ratings.</string>
+  <string name="ratedGame">Make sure you played a rated game. Casual games do not affect the players ratings.  In certain circumstances when playing against a bot account, a rated game may not award points if it is determined that the player is abusing the bot for rating points.</string>
+  <string name="botRatingAbuse">In certain circumstances when playing against a bot account, a rated game may not award points if it is determined that the player is abusing the bot for rating points.</string>
```

Comment:
duplicated text?

Reply:
Thanks for pointing that out.  Just removed it.